libs/components/ag-grid/package.json

```diff
@@ -35,8 +35,8 @@
     "@skyux/lookup": "0.0.0-PLACEHOLDER",
     "@skyux/popovers": "0.0.0-PLACEHOLDER",
     "@skyux/theme": "0.0.0-PLACEHOLDER",
-    "ag-grid-angular": "~31.2.0",
-    "ag-grid-community": "~31.2.0"
+    "ag-grid-angular": "~31.3.2",
+    "ag-grid-community": "~31.3.2"
```

Comment:
I see you've added a schematic to migrate folks to `31.3`, but I'm not seeing any breaking changes in this pull request. Would it be possible to support AG Grid `^31.2` (with the caret) without forcing folks to update? Once the fix is released for AG Grid 31 (hopefully) people would be automatically updated to the latest minor version after running `ng update`.

Reply:
Good call. Updated.